plugins/reporters/web-app-template/src/components/TableView.jsx

```diff
@@ -501,7 +519,7 @@ class TableView extends React.Component {
                         {
                             defaultPageSize: 100,
                             hideOnSinglePage: true,
-                            pageSizeOptions: ['50', '100', '250', '500', '1000', '5000'],
+                            pageSizeOptions: ['50', '100', '250', '500'],
```

Comment:
Please keep `, '1000', '5000'`; those were deliberately added in https://github.com/oss-review-toolkit/ort/pull/7144.

Reply:
Seems did got lost in whilst I was resolving a merge conflict in a rebase - fix it now.